our-work/software/vr-volume-viewer.yml

```diff
@@ -3,12 +3,14 @@ description: A simple viewer for visualizing volumetric datasets. It supports co
 languages:
   - c++
 tags:
-  - vr
+  - VR
```

Comment:
Remove MinVR. Revert VR to lower case (all tags are lower case at this point). I would change `Volume Rendering` for `visualization` and `3d`. I think we are trying to have tags general enough that they would include several cards if filtered by it. Any new tags that you add here, you'll need to check if they exist in the schema in order to pass the validation

Maybe also add `viz`

Reply:
`visualization` and `3d `  are good tags to include in this case. I'd add `multiplatform`